<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_building_list</class>
 <widget class="QWidget" name="widget_building_list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/frame/resources/frame/building1.png</pixmap>
   </property>
  </widget>
  <widget class="IconButton" name="tower1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="price1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="price2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="tower2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="price3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="tower3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="price4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="tower4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="name4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="IconButton" name="price0">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="tower0">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 4px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="name0">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>IconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
